src/lib.rs

```diff
@@ -29,8 +29,6 @@ pub mod note;
 /// Phoenix Core Keys & Addresses
 mod keys;
 
```

Comment:
If we want to use the hash function in other libraries downstream we might want to expor it here:
```
pub use keys::hash;
```